### ISSUE TITLE
Issue "Interlis Topic Filter" with side effect - ili2fme-6.2.3-20181209 & FME 2019.0.1.0 

### ISSUE THREAD

**Issue by behrm** [claeis/ili2fme#9]
I have Interlis data (DM01AVLV95SG2403.Gebaeudeadressen.Strassenplan) containing bad/erroneous polygon data (self-overlapping polygon etc). When I configure the Interlis-Reader to ignore this errors (ITF Check Polygon Building: No) the FME-workbench is processed normally and ends with "Translation was SUCCESSFUL".

If I additionally configure an Interlis Topic Filter (eg. Topics Filter: DM01AVLV95SG2403.Bodenbedeckung) to direct ili2fme to process only a subset of Interlis-data the following error is issued:

```
intersection tids 48, 48
intersection tids 48, 48
failed to build polygons of DM01AVLV95SG2403.Gebaeudeadressen.Strassenplan.Geometrie
  intersections
ch.interlis.iox_j.IoxInvalidDataException: failed to build polygons
	at ch.interlis.iom_j.itf.ItfReader2.read(ItfReader2.java:188)
	at ch.interlis.ili2fme.Ili2Reader.myread(Ili2Reader.java:717)
	at ch.interlis.ili2fme.Ili2Reader.read(Ili2Reader.java:633)

```
Apparently specifying a "Interlis Topic Filter" forces ili2fme to neglect the setting of "ITF Check Polygon Building: No"

To my knowledge the last Version which had this desireable feature (no polygon building check working together with topic-filter) was FME 2016.1.3.0 (20161103 - Build 16709 - WIN64) combined with ili2fme-5.8.0-20131209

**So in summary please allow topic filtering combined with disabled polygon building check**

**Comment by claeis** [comment]
duplicate of #1

**Comment by claeis** [comment]
I implemented the filter with the wrong logic
